MarketplaceService/Exceptions/EmptyFieldException.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace marketplaceservice.Exceptions
+{
+    [Serializable]
+    public class EmptyFieldException : Exception
+    {
+        public EmptyFieldException()
+            : base("All fields have to be filled out.")

```

Comment:
Filled out moet filled in zijn

Reply:
https://www.linguee.nl/engels-nederlands/vertaling/needs+to+be+filled+out.html